packages/Kokkos/src/DTK_LinearBVH.hpp

```diff
@@ -0,0 +1,80 @@
+#ifndef DTK_LINEAR_BVH_HPP
+#define DTK_LINEAR_BVH_HPP
+
+#include <Kokkos_ArithTraits.hpp>
+#include <Kokkos_Array.hpp>
+#include <Kokkos_Pair.hpp>
+#include <Kokkos_View.hpp>
+
+#include <Kokkos_OpenMP.hpp> // TMP
+
+namespace DataTransferKit
+{
+
+// Axis-Aligned Bounding Box
+// This is just a thin wrapper around an array of size 2x spatial dimension with
+// a default constructor to initialize properly an "empty" box.
+struct AABB
+{
+    using ArrayType = Kokkos::Array<double, 6>;
+    using SizeType = ArrayType::size_type;
```

Comment:
Sure.  I don't really care what convention we adopt.  I was following [our style guide on readthedocs](http://datatransferkit.readthedocs.io/en/latest/coding_guidelines.html#style-guide).